docs/how_to/enabling_access_to_vmpooler.md

```diff
@@ -0,0 +1,18 @@
+# Enabling access to the vmpooler VMs during an acceptance test run
+
+If you are running acceptance tests for Beaker that utilise vmpooler, you will need to run an SSH agent and add the acceptance RSA key prior to running the tests
+
+To load the SSH agent and add the acceptance key, run the following:
+
+~~~bash
+eval `ssh-agent`
+ssh-add ~/.ssh/id_rsa-acceptance
```

Comment:
So my understanding is that the id_rsa key is the private key everyone generates when they join Puppet, the public component of which gets added to puppetlabs-modules and puppetlabs-sshkeys to enable access to GitHub/puppetlabs machines - I think the Jenkins nodes have their own id_rsa keys for accessing repos on GitHub. And yep, I believe that the id_rsa-acceptance key provides access to the VMPooler VMs